HW2_Stepanova/calculator.py

```diff
@@ -0,0 +1,70 @@
+# функция, которая спрашивает, хочет ли человек продолжить расчет
+def end():
+    while True:
+        x = input('Вы хотите совершить еще один расчет?\nОтвет: да/нет ').lower()
+        if x == 'да':
+            return False
+            break
+        elif x == 'нет':
+            return True
+            break
+        else: 
+            print('Неподходящий ответ')
+
+# функция возвращает F или T, соотвественно, в цикле ее можно указать в виде if end():
+# if end():
+#     print('Конец программы')
+# else:
+#     print('Расчет дальше')
+
+def  get_numbers():
+    exp = (input("Введите выражение: "))
+    return exp
+
+def operation(expression):
+    a,oper,b=expression.split()
+    a=float(a)
+    b=float(b)
+    return(a,b,oper)
+
+
+def calculate(a, b, oper):
+    if oper == "+":
+        result = a + b
+    elif oper == "-":
+        result = a - b
+    elif oper == "*":
+        result = a * b
+    elif oper == "/":
+        if b != 0:
+            return a / b
+        else:
+            return "Ошибка: Деление на ноль!"
```

Comment:
в случаях, когда обе ветки в if/else заканчиваются `return`-ом, обычно "экономят отступы":
```suggestion
        if b != 0:
            return a / b
        return "Ошибка: Деление на ноль!"
```